.github/workflows/test.yml

```diff
@@ -12,12 +12,12 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
-        node-version: [12, '*']
+        node-version: [16, '*']
```

Comment:
This is causing it to not pass. Why is it needed? Doesn't `*` do version 16?

Reply:
this was me hitting all the buttons it is not needed